sdk/servicebus/azure-servicebus/azure/servicebus/exceptions.py

```diff
@@ -209,18 +210,22 @@ def on_connection_error(self, error):
         return super(_ServiceBusErrorPolicy, self).on_connection_error(error)
 
 
-class ServiceBusError(Exception):
-    """An error occurred.
-
-    This is the parent of all Service Bus errors and can
-    be used for default error handling.
+class ServiceBusError(AzureError):
+    """Base exception for all Service Bus errors which can be used for default error handling.
 
+    :param str message: The message object stringified as 'message' attribute
+    :keyword error: The original exception if any
+    :paramtype error: Exception
+    :ivar exc_type: The exc_type from sys.exc_info()
+    :ivar exc_value: The exc_value from sys.exc_info()
+    :ivar exc_traceback: The exc_traceback from sys.exc_info()
+    :ivar exc_msg: A string formatting of message parameter, exc_type and exc_value
+    :ivar str message: A stringified version of the message parameter
     """
 
-    def __init__(self, message, inner_exception=None):
-        # type: (Optional[str], Optional[Exception]) -> None
-        self.inner_exception = inner_exception
-        super(ServiceBusError, self).__init__(message)
+    def __init__(self, message, *args, **kwargs):
+        # type: (str, Any, Any) -> None
+        super(ServiceBusError, self).__init__(message, error=kwargs.get("error"))
```

Comment:
Why do we have `*args` if we just ignore it?
Also - would there be any issues with just passing all `kwargs` down to `super`?

Reply:
I think you're right, we could just pass args and kwargs through as it's something AzureError supports.